content/docs/1.7.0/advanced-resources/backing-image/backing-image-backup.md

```diff
@@ -75,8 +83,24 @@ spec:
 >   - `concurrent-limit`: Maximum number of worker threads that can concurrently run for each restore operation. When unspecified, Longhorn uses the default value.
 > - `checksum`: You can specify the expected SHA-512 checksum of the backing image file, which Longhorn uses to validate the restored file. When unspecified, Longhorn uses the checksum of the restored file as the truth.
 
+Restore backing image from a backup via UI
+1. Go to `Setting -> Backing Image` page.
+2. Click the `Restore` action in operation menu for the backup backing image you want to restore
+3. Input the `Secret` and `Secret Namespace` if you want to restore an encrypted backup backing image
```

Comment:
I think this is more accurate.

`3. Input the Secret and Secret Namespace if the backing image is an encrypted image`